src/reflect/scala/reflect/internal/Printers.scala

```diff
@@ -1054,13 +1054,13 @@ trait Printers extends api.Printers { self: SymbolTable =>
 
         case Literal(k @ Constant(s: String)) if s.contains(Chars.LF) =>
           val tq = "\"" * 3
-          val lines = s.lines.toList
-          if (lines.lengthCompare(1) <= 0) print(k.escapedStringValue)
+          val lines = s.linesIterator
+          if (lines.hasNext) print(k.escapedStringValue)
```

Comment:
That's not the same, is it? `hasNext` means `size >= 1`; this should be `size <= 1`. I think `toList` is justifiable.